prboom2/src/r_bsp.c

```diff
@@ -794,12 +809,7 @@ static void R_Subsector(int num)
 
   if (sub->sector->validcount != validcount)
   {
-    sub->sector->validcount = validcount;
```

Comment:
Why is this line gone? Does this mean we'll be calling R_AddSprites more than we should?

Reply:
I'll refactor this a bit to avoid messing with validcount in multiple places